source/extensions/common/wasm/context.h

```diff
@@ -458,6 +464,11 @@ class Context : public Logger::Loggable<Logger::Id::wasm>,
   std::map<uint32_t, AsyncClientHandler> http_request_;
   std::map<uint32_t, GrpcCallClientHandler> grpc_call_request_;
   std::map<uint32_t, GrpcStreamClientHandler> grpc_stream_;
+
+  // Evaluator state
+  Filters::Common::Expr::BuilderPtr builder_{};
+  uint32_t next_expr_token_ = 0;
+  std::map<uint32_t, Filters::Common::Expr::ExpressionPtr> expr_;
```

Comment:
if we are using absl stuff, we can use absl::flat_hash_map here?
It is said to be more perform-ant then std::map..

Reply:
The code here is very opinionated, so I'm trying to conform to the existing style. Other places use map.